git-hub

```diff
@@ -1849,19 +1862,24 @@ class RebaseCmd (PullUtil):
     # Performs the rebasing itself. Sets the `in_conflict` flag if
     # a conflict is detected.
     @classmethod
-    def rebase(cls, args, pull):
+    def rebase(cls, args, pull, is_merge, base_hash, head_hash):
         starting = args.action is None
         try:
             if starting:
+                # Last commit is a merge commit, so ask the user to merge
```

Comment:
yeah it is here, but this comment is outdated then: https://github.com/sociomantic-tsunami/git-hub/pull/183/files#diff-88fb15e32f4101f90004ef18b66b704bR1800